test/validate/eia_test.py

```diff
@@ -44,7 +44,7 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
         ("bga_eia860", 130_326, 130_326, 130_326),
         ("boil_eia860", 74_086, 74_086, 74_086),
         ("frc_eia923", 597_000, 244_415, 24_065),
-        ("gen_eia923", None, 5_171_497, 432_570),
+        ("gen_eia923", None, 4_872_421, 407_653),
```

Comment:
Why did the number of records change?

Reply:
IIRC this (substantial) change is because I got rid of the last year of records, which had only null data in it since the only 2022 data we have is from EIA-860M, and these outputs depend on data tables from EIA-923.